cogs/events.py

```diff
@@ -43,6 +43,19 @@ async def on_command_error(self, ctx, error):
             em = discord.Embed(color = discord.Color.red(),
                                description = f"`{str(error.param).partition(':')[0]}` is a required argument!")
             await ctx.send(embed = em)
+                               
+    @commands.Cog.listener('on_command_error')
```

Comment:
Why do this when you could make the function name `on_command_error`

Reply:
It would clash with the error handler directly above and I think its good to split the error logs from the error messages